ocs_ci/ocs/resources/pod.py

```diff
@@ -143,6 +145,58 @@ def exec_ceph_cmd(self, ceph_cmd, format='json-pretty'):
             return [item for item in out if item]
         return out
 
+    def run_io(
```

Comment:
My understanding is that the run_io() will, in future, be used for invoking the workloads io for Couchbase, MongoDB also. Is that correct? If yes, then the parameters that it accepts currently is fio specific. Should that be made generic or should we expose execute_fio?

Reply:
Ru_io is meant to run IO (at this point).
If we will see we need to use it for invoking IO for different workloads in the future, it can be easily changed